database-api/src/main/kotlin/fi/vauhtijuoksu/vauhtijuoksuapi/database/api/VauhtijuoksuDatabase.kt

```diff
@@ -23,6 +24,13 @@ interface VauhtijuoksuDatabase<T : Model> {
      * Update given record and return the updated record if it exists. Note that full record is expected as input,
      * i.e. any null values are written to db.
      */
+
+    fun addAll(records: List<T>): Future<List<T>?> { throw ServerError("Missing addAll implementation") }
```

Comment:
The docstrings are wrong way around. Also remove the empty line between doc and method name

Reply:
swapped